sportnet/sportnet/playgrounds/views.py

```diff
@@ -14,3 +15,18 @@ def get(self, request, *args, **kwargs):
         content = self.get_context_data(**kwargs)
         content['form'] = form
         return self.render_to_response(content)
+
+
+class UserIsAdmin(UserPassesTestMixin):
+    def test_func(self):
+        qs = Event.objects.filter(id=self.kwargs['pk'])
+        return qs.filter(admins=self.request.user)
+
+
+class PlaygroundsDetailView(DetailView):
+    model = Event
+
+
+class PlaygroundsDeleteView(LoginRequiredMixin, UserIsAdmin, DeleteView):
+    model = Event
+    success_url = 'playgrounds/event_confirm_delete'
```

Comment:
```suggestion
    success_url = reverse_lazy('playgrounds:index')
```
Так, после удаления перекинет на главную страницу. До конца не понял почему, но когда указываешь именно параметр класса, нужно указывать `reverse_lazy` (а не обычный `reverse`). Хардкодить в ручную, всегда плохой вариант сам по себе.
У тебя сейчас просто указан левый не существующий адрес.